src/ApiClient/index.tsx

```diff
@@ -4,7 +4,7 @@
 import { NativeModules } from "react-native";
 import isURL from "validator/es/lib/isURL";
 
-const { RNNCAPIClient } = NativeModules;
+const { APIClient: RNNCAPIClient } = NativeModules;
```

Comment:
Omg, I've seen this changing over and over again. Can you and @migbot come to a decision and stick by it??

I would love to have this resolved asap